api.go

```diff
@@ -121,3 +138,9 @@ select json_arrayagg(json_object(
 from info i
 `,
 }
+
+func (a *api) error(s *server, err error) {
```

Comment:
nit: move this up with the rest of the api methods, so that the top-half of the file is the generic API handling code, and the bottom-half is the list of specific API implementations